### ISSUE TITLE
Mac-made .csv files won't parse

### ISSUE THREAD

**Issue by gCurtisCT** [Keyang/node-csvtojson#55]
When creating a .csv generated by the OSX version of Excel, you have to save the file specifically as "Windows Commas Separated" .csv file to get it to parse, otherwise, the converter will return an empty array.

When saving it as the normal, "default" .csv type, it won't parse.

```
var Converter		= require('csvtojson').Converter;
var converter = new Converter({constructResult:true});
converter.on("end_parsed",function(json){
	if(json.length>0){
		report.json = json;
		defer.resolve(json);
	}else{
        //this happens when a mac .csv is sent
		defer.reject({message:'.csv could not be parsed. Please create a "Windows Comma Separated" .csv file'});
	}
});
fs.createReadStream(fileLocation).pipe(converter);
```

**Comment by Keyang** [comment]
Could you let me know what version of csvtojson you are using?

~Keyang

**Comment by gCurtisCT** [comment]
0.4.1

**Comment by Keyang** [comment]
I think it is related to #53 which is fixed in 0.4.2

**Comment by gCurtisCT** [comment]
Yep, it is.
